### PR TITLE
Most basic display of how to revert to a previous PR

### PR DESCRIPTION
**Pull request by ausmith**
This does not have tests and does not necessarily look good. Feel free to pull in these changes though and style as you wish. This will provide your command to revert to a specific PR in the production branch from any of our developer's local machines.

screen shot 2016-05-26 at 12 23 47 pm

### CODE REVIEW

**Comment by oochelz** [comment]
So this is if I need to rollback, I should use that? Or when I want to revert the rollback?
I'm not comfortable with this as is, because it isn't clear what it's for.

**Comment by ausmith** [comment]
The command is to go to the point that the specified PR is at. As indicated in the description, this could use some slightly improved verbiage/styling to make that clearer.

**Comment by ausmith** [comment]
Essentially saying "I want production to go to THIS point in its history".

File: app/views/deployments/list.html.haml [hunk 1]
```diff
@@ -25,4 +25,10 @@
     .small-12.large-6.columns.deployment-message
       = link_to pull.title, deployments_details_path(pull_id: pull.pull_number)
     .small-12.large-3.columns.deployment-time
-      = pull.merged_at
\ No newline at end of file
+      = pull.merged_at
+  .row.commit
+    .small-12.large-3.columns
+      Revert to PR ##{ pull.pull_number }
```
Comment (oochelz) [inline app/views/deployments/list.html.haml]: "To rollback to this deployment, use the following command:"

**Comment by oochelz** [comment]
I know I can work on styling differently later. But having something no one understands is arguably not much better than not having it at all, you know? That's why I care about the way it's worded, at least.

File: app/views/deployments/list.html.haml (hunk 1 above)
Comment (oochelz) [inline app/views/deployments/list.html.haml]: "redeploy" instead of "rollback"

Comment (ausmith) [inline app/views/deployments/list.html.haml]: If that's the wording that makes sense to you. Makes less sense to me as it doesn't indicate the PR that you are going back to.

**Comment by ausmith** [comment]
This PR is not required to be merged as is. It is simply proposing a solution that can be adapted for what makes the most sense for your team.

**Comment by ausmith** [comment]
Verbiage updated, screenshot reflects that.

**Comment by ausmith** [comment]
PR switched to a rollback.

**Comment by oochelz** [comment]
Ok, so this is to rollback that deployment, and not have us in a screwed up state where it's difficult to redeploy, right?

**Comment by oochelz** [comment]
That makes sense to me.

**Comment by usbsnowcrash** [comment]
Merged it into my own branch https://github.com/usbsnowcrash/ruby-deployments-dashboard/pull/34

**Pull request closed without merging**